lib/cc/pull_requests.rb

```diff
@@ -1,8 +1,15 @@
 class CC::PullRequests < CC::Service
+  VALID_TOKEN_MESSAGE = "Access token is valid.".freeze
+  CANT_UPDATE_STATUS_MESSAGE = "Access token is invalid - can't update status.".freeze
+
   def receive_test
     setup_http
 
-    receive_test_status
+    if able_to_update_status?
+      { ok: true, message: VALID_TOKEN_MESSAGE }
+    else
+      { ok: false, message: CANT_UPDATE_STATUS_MESSAGE }
+    end
```

Comment:
The test status is dependent on two different HTTP requests. Are you suggesting we should only show the first one that failed? Or combine them if they both failed?

I think the higher level messaging is actually useful: we're telling the user directly that this token needs to be able to set commit statuses, and also post comments. Error messages straight from the source are also helpful, of course. Maybe we should combine these and make the message strings format strings we can insert the HTTP response message into.